{"kind":"symbol","schemaVersion":{"major":0,"patch":0,"minor":3},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"languages":["swift"]}]}],"abstract":[{"type":"text","text":"Inherited from "},{"code":"Error.localizedDescription","type":"codeVoice"},{"type":"text","text":"."}],"hierarchy":{"paths":[["doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/exception\/localizeddescription"]}],"identifier":{"url":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","interfaceLanguage":"swift"},"sections":[],"metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"localizedDescription"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"}],"extendedModule":"Swift","title":"localizedDescription","symbolKind":"property","platforms":[{"unavailable":false,"beta":false,"deprecated":false,"introducedAt":"8.0","name":"iOS"},{"unavailable":false,"beta":false,"deprecated":false,"introducedAt":"10.10","name":"macOS"},{"unavailable":false,"beta":false,"deprecated":false,"introducedAt":"9.0","name":"tvOS"},{"unavailable":false,"beta":false,"deprecated":false,"introducedAt":"2.0","name":"watchOS"}],"role":"symbol","externalID":"s:s5ErrorP10FoundationE20localizedDescriptionSSvp::SYNTHESIZED::s:10SwiftKyber9ExceptionO","roleHeading":"Instance Property","modules":[{"name":"SwiftKyber","relatedModules":["Swift"]}]},"references":{"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber":{"title":"SwiftKyber","url":"\/documentation\/swiftkyber","kind":"symbol","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","type":"topic","abstract":[{"text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard","type":"text"}],"role":"collection"},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/localizedDescription":{"kind":"symbol","title":"localizedDescription","abstract":[],"url":"\/documentation\/swiftkyber\/exception\/localizeddescription","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"localizedDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"}],"type":"topic","role":"symbol"},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception":{"role":"symbol","title":"Exception","kind":"symbol","type":"topic","abstract":[{"type":"text","text":"The Kyber exceptions"}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Exception","kind":"identifier"}],"url":"\/documentation\/swiftkyber\/exception","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","navigatorTitle":[{"text":"Exception","kind":"identifier"}]}}}